### PR TITLE
Fix message passage for long commands, add more FAQs

### PR DESCRIPTION
**Pull request by Sbozzolo**
@condy0919 I added your PR #320 to the FAQ, what do you think?

### CODE REVIEW

**Comment by condy0919** [comment]
Fine with me

File: README.md [hunk 1]
```diff
@@ -594,6 +543,28 @@ the correct function to yank in vterm buffers.
...
+tracking. If you cannot do it, a possible workaround is the following.
+
+On most GNU/Linux systems, you can read current directory from `/proc`:
+```emacs-lisp
+(defun vterm-directory-sync ()
+  "Synchronize current working directory."
+  (interactive)
+  (when vterm--process
+    (let* ((pid (process-id vterm--process))
+           (dir (file-truename (format "/proc/%d/cwd" pid))))
```
Comment (condy0919) [inline README.md:558]: I found `(dir (file-truename (format "/proc/%d/cwd/" pid))))` is better

Comment (Sbozzolo) [reply README.md:558]: That's what we have here. Am I missing something?

Comment (condy0919) [reply README.md:558]: `/proc/%d/cwd/` with an additional `/` makes it prettier in `counsel`.

no trialing slash

prettier

File: README.md [hunk 2]
```diff
@@ -594,6 +543,28 @@ the correct function to yank in vterm buffers.
...
+      (setq-local default-directory dir))))
```
Comment (brotzeit) [inline README.md:559]: I think `default-directory` becomes automatically buffer local.

Comment (Sbozzolo) [reply README.md]: Thanks

File: README.md (hunk 1 above)
Comment (Sbozzolo) [reply README.md]: Thanks for the clarification

**Pull request merged**